beacon-chain/core/helpers/sync_committee.go

```diff
@@ -44,7 +45,7 @@ func IsCurrentPeriodSyncCommittee(
 
 		// Fill in the cache on miss.
 		go func() {
-			if err := syncCommitteeCache.UpdatePositionsInCommittee(bytesutil.ToBytes32(root), st); err != nil {
+			if err := syncCommitteeCache.UpdatePositionsInCommittee(root, st); err != nil {
```

Comment:
Are you sure you don't need to use `combineRootAndSlot` for `UpdatePositionsInCommittee` here?

Reply:
Ah sorry. The diff didn't show that